src/DurableTask.AzureStorage/AzureStorageOrchestrationService.cs

```diff
@@ -1106,6 +1106,11 @@ async Task AbandonAndReleaseSessionAsync(OrchestrationSession session)
             TaskMessage continuedAsNewMessage,
             OrchestrationState orchestrationState)
         {
+            foreach (var timerMessage in timerMessages)
+            {
+                Utils.ConvertTimeToUtc(timerMessage.Event);
+            }   
+
```

Comment:
nit - would like to document this change so we know why this is DTFx.AS-specific
```suggestion
        {
            // for backwards compatibility, we transform timestamps to UTC prior to persisting in AzStorage.
            // see: https://github.com/Azure/durabletask/pull/1138
            foreach (var timerMessage in timerMessages)
            {
                Utils.ConvertTimeToUtc(timerMessage.Event);
            }   

```